### DISCUSSION TITLE
Unable to create first user

### DISCUSSION THREAD

**Discussion by grundiss** [payloadcms/payload#1208] [Q&A]
# Bug Report

After upgrading payload from 1.0.27 to 1.1.4 I'm no longer able to create first user when use clean DB.

```
[👽] [08:31:55] ERROR (payload): TypeError: Cannot read properties of undefined (reading 'config')
[👽]     at %project%/node_modules/payload/dist/fields/validations.js:278:9
[👽]     at Array.filter (<anonymous>)
[👽]     at relationship (%project%/node_modules/payload/src/fields/validations.ts:320:4)
[👽]     at Object.field.validate (%project%/node_modules/payload/src/fields/config/sanitize.ts:56:46)
[👽]     at promise (%project%/node_modules/payload/src/fields/hooks/beforeChange/promise.ts:107:44)
[👽]     at processTicksAndRejections (node:internal/process/task_queues:96:5)
[👽]     at async Promise.all (index 3)
[👽]     at traverseFields (%project%/node_modules/payload/src/fields/hooks/beforeChange/traverseFields.ts:60:3)
[👽]     at beforeChange (%project%/node_modules/payload/src/fields/hooks/beforeChange/index.ts:34:3)
[👽]     at create (%project%/node_modules/payload/src/collections/operations/create.ts:151:29)
```

I debugged that file a bit and heres' what my console displays:

```
[👽] {
[👽]   'options.payload.collections': {
[👽]     users: { ... },
[👽]     sites: { ... },
[👽]     companies: { ... },
[👽]     technologies: { ... },
[👽]     'blog-posts': { ... },
[👽]     media: { ... }
[👽]   },
[👽]   collection: undefined,
[👽]   'options.payload.collections[collection]': undefined,
[👽]   val: { value: undefined, relationTo: undefined },
[👽]   values: [ { value: undefined, relationTo: undefined } ],
[👽] }

```

## Current Behavior

First user cannot get created

## Expected Behavior

First user gets created

**Comment by grundiss** [comment]
The problem does not appear if I switch back to 1.0.27

**Comment by Rar9** [comment]
Same here 😢 
 I get Cors a issue  
`  Cross-Origin Request Blocked: The Same Origin Policy disallows reading the remote resource at http://localhost:3400/api/users/me`

**Comment by Rar9** [comment]
According to the Mozzila Documentation - Reason: the CORS request did not succeed:
Possible causes include:

Trying to access an https resource that has an invalid certificate will cause this error.
Trying to access an http resource from a page with an https origin will also cause this error.

As of Firefox 68, https pages are not permitted to access http://localhost

Also, the error differs from Chrome, where it shows(images attached):

Since my the proxy_pass configured is working correctly and ports are correctly open:

The issue appears to be related to the URL http://localhost:3400/api/users/me and http://localhost:3400/api/users/first-register being unable to be resolved, and the error NetworkError when attempting to fetch resource is also observed.

**Comment by jmikrut** [comment]
Hey @grundiss — I can't reproduce your issue, unfortunately. I think this is probably a configuration issue as spinning up a new Payload app with `create-payload-app` works as intended. I am going to convert this to a discussion but am happy to help you troubleshoot.

What would be helpful is if you were able to spin up a minimally reproducible demo so that I can help you test and recreate the issue. 

@Rar9 — your issue is separate, and I believe yours also has to do with your specific configuration. I am going to help you out on Discord, so I'll chat with you there!

**Reply by Rar9** [reply]
yes config issues...thanks
